tools/roxie/extract-roxie-timings.py

```diff
@@ -104,6 +106,9 @@ def printRow(curRow):
                 mapping = rowText.split();
 
                 serviceName = completeMatch.group(1)
+                if ignoreQueryCase:
```

Comment:
Or could add re.IGNORECASE to re.compile("COMPLETE") at line 75 above
Or could use .lower() in if not serviceName in allServices: at line 184 below
Or even some other ways, never sure which is best.